src/wiseagents/graphdb/lang_chain_wise_agent_graph_db.py

```diff
@@ -36,7 +38,9 @@ def __init__(self, embedding_model_name: Optional[str] = DEFAULT_EMBEDDING_MODEL
             embedding_model_name (Optional[str]): the optional name of the embedding model to use 
         """
         self._embedding_model_name = embedding_model_name
-        self._embedding_function = HuggingFaceEmbeddings(model_name=self.embedding_model_name)
+        model_kwargs = dict()
+        model_kwargs['tokenizer_kwargs']={"clean_up_tokenization_spaces": True}
```

Comment:
Shouldn't this be False too?

Reply:
I kep the current behaviour as far as I understood the warning: clean_up_tokenization_spaces was not set. It will be set to True by default. This behavior will be depracted in transformers v4.45, and will be then set to False by default.